js/views/itemEditVw.js

```diff
@@ -204,13 +210,26 @@ module.exports = Backbone.View.extend({
   },
 
   resizeImage: function(){
-    "use strict";
     var self = this,
-        imageFiles = this.$el.find('.js-itemImageUpload')[0].files,
+        $imageInput = this.$el.find('.js-itemImageUpload'),
+        imageFiles = Array.prototype.slice.call($imageInput[0].files, 0),
+        curImages = this.model.get('combinedImagesArray'),
         maxH = 357,
         maxW = 357,
         imageList = [],
-        imageCount = imageFiles.length;
+        loaded = 0,
+        imageCount;
+
+    $imageInput.val('');
```

Comment:
Another obscure bug fix here... am resetting the value of the file input, because if you don't the change event won't fire on a subsequent upload if you a.) upload the same file that was previously uploaded b.) upload a batch of files, with the last one being the same as the one previously uploaded c.) upload a file that is the same as the last of a batch previously uploaded.